src/crimson/os/alienstore/alien_store.cc

```diff
@@ -56,17 +59,16 @@ class OnCommit final: public Context
 
 namespace crimson::os {
 
+using crimson::common::get_conf;
+
 AlienStore::AlienStore(const std::string& path, const ConfigValues& values)
   : path{path}
 {
   cct = std::make_unique<CephContext>(CEPH_ENTITY_TYPE_OSD);
```

Comment:
if cct is not used to get configure, we can remove cct initialized in this function.

Reply:
i think the reason why we have `cct` in `AlienStore` is to ensure that `g_ceph_context` is referencing a valid `CephContext` while the `AlienStore` instance is alive. because `g_ceph_context` does not hold a strong reference to it, we have to keep a strong reference somewhere else. in this case, we let `AlienStore` to hold it.